app/components/work_history_review_component.rb

```diff
@@ -101,4 +111,16 @@ def formatted_end_date(work)
 
     work.end_date.to_s(:month_and_year)
   end
+
+  def generate_id(work_id:, attribute:)
+    "work-history-#{work_id}-#{attribute}"
+  end
```

Comment:
I'm wondering if this method is really necessary; thinking out loud:

```ruby
generate_id(13, 'dates')
```

```ruby
"work-history-13-dates"
```

We need to duplicate the `generate_id` call anyway in this file, the code it's replacing isn't too complex. There's the argument that it does DRY the `work-history-` bit but ehhh... it's not a huge difference either way.

Feel free to ignore this ramble. 👍

Reply:
So I like the idea that generating an `id` is handled by a single method so if we wanted it to change the structure we just need to update that method I believe. Also keeping an eye on the fact some refactoring may be done as other sections have the same problem with the `Change` links.

Am going to leave it how it is, although whilst revising this I did notice I didn't need the variable assignments e.g. `id = generate_id(work_id: work.id, attribute: 'dates')` so updating this.